File: test-suite/handwritten-src/java/com/dropbox/djinni/test/SimpleObjectImpl.java
package com.dropbox.djinni.test;

public class SimpleObjectImpl extends SimpleObject {
    private final int value;
    private final String name;

    public SimpleObjectImpl(int value, String name) {
        this.value = value;
        this.name = name;
    }

    @Override
    public int getValue() {
        return value;
    }

    @Override
    public String getName() {
        return name;
    }
}

File: test-suite/handwritten-src/swift/SimpleObjectImpl.swift
import Foundation
@testable import TestSuite

class SimpleObjectImpl: SimpleObject {
    private let value: Int32
    private let name: String
    
    init(value: Int32, name: String) {
        self.value = value
        self.name = name
    }
    
    func getValue() throws -> Int32 {
        return value
    }
    
    func getName() throws -> String {
        return name
    }
}
